packages/dotenv-webpack-plugin/src/index.js

```diff
@@ -0,0 +1,68 @@
+const fs = require("fs");
+const path = require("path");
+const dotenv = require("dotenv");
+const dotenvExpand = require("dotenv-expand");
+const { DefinePlugin } = require("webpack");
+
+class DotenvWebpackPlugin {
+  constructor(config = {}) {
+    this.config = Object.assign(
+      {},
+      {
+        envFiles: [],
+        prefixes: ["process.env.", "import.meta.env."],
+      },
+      config,
+    );
+    this.cache = new Map();
```

Comment:
Avoid it right now, we should use webpack cache API, but let's do it when our code will be finished, cache is the last step